### PR TITLE
DOCS-535: Markup fixes. Some remarks about formatting for Nested structures

### PR DESCRIPTION
**Pull request by BayoNet**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Documentation

### CODE REVIEW

File: docs/en/interfaces/formats.md [hunk 1]
```diff
@@ -99,10 +99,34 @@ The minimum set of characters that you need to escape when passing data in TabSe
 
 Only a small set of symbols are escaped. You can easily stumble onto a string value that your terminal will ruin in output.
 
-Arrays are written as a list of comma-separated values in square brackets. Number items in the array are fomratted as normally, but dates, dates with times, and strings are written in single quotes with the same escaping rules as above.
+Arrays are written as a list of comma-separated values in square brackets. Number items in the array are formatted as normally, but dates, dates with times, and strings are written in single quotes with the same escaping rules as above.
```
Comment (qoega) [inline docs/en/interfaces/formats.md]:
```suggestion
Arrays are written as a list of comma-separated values in square brackets. Number items in the array are formatted as normally. `Date` and `DateTime` types are written in single quotes. Strings are written in single quotes with the same escaping rules as above.
```

File: docs/en/interfaces/formats.md [hunk 2]
```diff
@@ -99,10 +99,34 @@ The minimum set of characters that you need to escape when passing data in TabSe
...
+
+For example:
+
+```sql
+CREATE TABLE nestedt
+(
+    `id` UInt8, 
+    `aux` Nested(
+    a UInt8, 
+    b String)
```
Comment (qoega) [inline docs/en/interfaces/formats.md]: more ident here?

File: docs/en/interfaces/formats.md [hunk 3]
```diff
@@ -99,10 +99,34 @@ The minimum set of characters that you need to escape when passing data in TabSe
 
 Only a small set of symbols are escaped. You can easily stumble onto a string value that your terminal will ruin in output.
 
-Arrays are written as a list of comma-separated values in square brackets. Number items in the array are fomratted as normally, but dates, dates with times, and strings are written in single quotes with the same escaping rules as above.
+Arrays are written as a list of comma-separated values in square brackets. Number items in the array are formatted as normally, but dates, dates with times, and strings are written in single quotes with the same escaping rules as above.
 
 [NULL](../query_language/syntax.md) is formatted as `\N`.
 
+Elements of [Nested](../data_types/nested_data_structures/nested.md) structures are formatted as arrays each.
```
Comment (qoega) [inline docs/en/interfaces/formats.md]: Is is not only a format issue. Array of structures is converted to arrays of nested fields

**Review by qoega — CHANGES_REQUESTED** [review]

File: docs/en/interfaces/formats.md (hunk 2 above)
Comment (BayoNet) [reply docs/en/interfaces/formats.md]: Yes.

**Pull request merged**